widget/entry.go

```diff
@@ -2308,12 +2305,14 @@ func (i *entryModifyAction) Redo(s string) string {
 
 // Inserts Text
 func (i *entryModifyAction) add(s string) string {
```

Comment:
We have casting to strings that result in allocations here and in `sub` now. Would refactoring these to receive and return []rune result in fewer castings still?

Reply:
It looks like that opens a can of worms. as the undo/redo stack passes it up and down, with the source being `Entry.Text` which is `string`. I agree it would be a good improvement but probably could be another PR.